homeassistant/components/alarm_control_panel/const.py

```diff
@@ -3,12 +3,15 @@
 SUPPORT_ALARM_ARM_HOME = 1
 SUPPORT_ALARM_ARM_AWAY = 2
 SUPPORT_ALARM_ARM_NIGHT = 4
-SUPPORT_ALARM_TRIGGER = 8
-SUPPORT_ALARM_ARM_CUSTOM_BYPASS = 16
+SUPPORT_ALARM_ARM_VACATION = 8
```

Comment:
Let's not change the existing support integers. Just add the new modes 32.

Reply:
Fixed